scss/civicrm/common/_forms.scss

```diff
@@ -463,6 +470,13 @@
         > div.select2-container {
           @extend %responsive;
           margin-right: 0 !important;
+
+          &.six,
+          &.eight,
+          &.crm-form-time {
```

Comment:
you are repeating this style 3 times, there must be a way to define it only once

Reply:
We are using different selectors here. To make it common, we can create a mixin and load at here.
Should I create one?

Reply:
yes please, and make sure to document it. If you could follow http://sassdoc.com/annotations/ for the documentation it would be great